app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(pleyers: list) -> int:
+    result = 0
+    for pleyer in pleyers:
+        result += pleyer.get_rating()
+    return result
+
+
+def elves_concert(elves: list) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarfs: list) -> None:
+    for dwarf in dwarfs:
+        dwarf.eat_favourite_dish()
```

Comment:
add more descriptive type annotations

Reply:
What's exactly means? For **pleyers, elves, dwarfs** "list" annotation is correct. The 1-st function returns an integer, 2-nd and 3-rd - nothing. Or for 2-nd and 3-rd functions "str" could be better annotation?

Reply:
This means that u should add smth like this list[<type inside list>] - more descriptivve type annotations

Reply:
also players - not pleyers